Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,74 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  resource "envman" do
+    url "https://github.com/bitrise-io/envman/archive/0.9.3.tar.gz"
+    sha256 "ff9cb03f978332499bad52f4930f736ded2f8573e8b713b5082f65432e618b8f"
+  end
+
+  resource "stepman" do
+    url "https://github.com/bitrise-io/stepman/archive/0.9.7.tar.gz"
+    sha256 "1eae33616e9c1c952f7c3cfc677fecc0cf0f0781a4aa2e686969e74485284c89"
+  end
+
+  def install
+    go_install_package = lambda do |basepth, go_packagename|
+      mkdir_p "#{basepth}/src/github.com/bitrise-io"
+      ln_s basepth, "#{basepth}/src/github.com/bitrise-io/#{go_packagename}"
+
+      ENV["GOPATH"] = "#{basepth}/Godeps/_workspace:#{basepth}"
+      Language::Go.stage_deps resources, "#{basepth}/src"
+      system "go", "build", "-o", "#{bin}/#{go_packagename}"
+    end
```

Comment:
I thought it's better to have this inside `install` as it's not used anywhere else, and that way for a brew Formula noob like myself it's probably more obvious what's required in a Formula and what's just utility code.